tests/cli/test_export_model.py

```diff
@@ -108,3 +110,41 @@ def test_reexport(monkeypatch, tmp_path):
     export_model(model_loaded, "exported_new.pt")
 
     assert Path("exported_new.pt").is_file()
+
+
+def test_private_huggingface(monkeypatch, tmp_path):
```

Comment:
This will fail for people running the tests locally, unless they have access to the repo & a token set, right? 

Could we skip the test if `HUGGINGFACE_TOKEN_METATRAIN` is unset? And set it on CI/locally as needed. (using a different name in case people are already using `HUGGINGFACE_TOKEN` for some other reason.